board/model_manager.py

```diff
@@ -105,11 +115,14 @@ def sample_data_and_train(self):
 
         self.logger.info("############ instantiating Samplers.. ############")
 
-        
-
-        train_sampler = RandomPairwiseSampler(batch_size=1000, 
-                                            dataset=train_dataset, 
-                                            num_process=5)
+        if self.recommender_name=="pmf" or self.recommender_name=="drr":
```

Comment:
This will be removed lated based on which samplers CANNOT work for which model algorithms

Reply:
you forgot to change: https://github.com/mohit-chawla/recboard/blob/fac90b0b22be06ee861b05fb3de77cbac4006780/board/templates/home.html#L314

Reply:
No i think it is fine. I have been doing it for `total_iter: 1000, eval_iter: 10, save_iter: 10` to observe training.